File: src/app/telemetry/FileIntegrity.java
package app.telemetry;

import java.io.File;
import java.io.IOException;
import java.net.URL;
import java.util.Scanner;

public class FileIntegrity {
  private static URL fileList;

  public FileIntegrity() {
    fileList = getClass().getResource("/files.txt");
  }

  public boolean isGood() throws IOException {
    Scanner sc = new Scanner(fileList.openStream());
    while (sc.hasNextLine()) {
      if (!new File("./resource/" + sc.nextLine()).exists()) {
        sc.close();
        return false;
      }
    }
    sc.close();
    return true;
  }

  public static String getAllFiles() throws IOException {
    Scanner sc = new Scanner(fileList.openStream());
    StringBuilder sb = new StringBuilder();
    while (sc.hasNextLine()) {
      sb.append(sc.nextLine()).append("\n");
    }
    sc.close();
    return sb.toString();
  }
}

File: kt/app/telemetry/FileIntegrity.kt
package app.telemetry

import app.telemetry.API
import kotlin.Throws
import java.io.IOException
import java.io.BufferedWriter
import java.io.FileWriter
import java.lang.StringBuilder
import app.telemetry.FileIntegrity
import app.CLI
import app.global.cli.CliType
import java.io.File
import java.net.URL
import java.util.*

class FileIntegrity {
    @get:Throws(IOException::class)
    val isGood: String
        get() {
            val sc = Scanner(fileList.openStream())
            while (sc.hasNextLine()) {
                val str = sc.nextLine()
                if (!File("./resource/" + sc.nextLine()).exists()) {
                    CLI.print("NOT FOUND: " + File("./resource/$str").name, CliType.ERROR)
                    sc.close()
                    return str
                }
                CLI.print("FOUND: " + File("./resource/$str").name, CliType.SUCCESS)
            }
            sc.close()
            return "0"
        }

    companion object {
        private lateinit var fileList: URL

        @get:Throws(IOException::class)
        val allFiles: String
            get() {
                val sc = Scanner(fileList.openStream())
                val sb = StringBuilder()
                while (sc.hasNextLine()) {
                    sb.append(sc.nextLine()).append("\n")
                }
                sc.close()
                return sb.toString()
            }
    }

    init {
        fileList = javaClass.getResource("/files.txt")
    }
}